src/graph_compute/phase_scheduling.h

```diff
@@ -20,13 +19,23 @@ typedef struct chunkdata_t chunkdata_t;
 
 struct scheddata_t {
   vid_t * dependentEdges;
-  vid_t * dependentEdgeIndex;
-  vid_t * cntDependentEdges;
+//  vid_t * dependentEdgeIndex;
+//  vid_t * cntDependentEdges;
   chunkdata_t * chunkdata;
   vid_t cntChunks;
 };
 typedef struct scheddata_t scheddata_t;
 
+struct sched_t {
+  vid_t * dependentEdges;
+  vid_t cntDependentEdges;
+  vid_t dependencies;
```

Comment:
Would be nice to add comments explaining what each of these fields does, for the sake of other people reviewing our code.